src/dash-table/components/CellMarkdown/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, {
+    PureComponent
+} from 'react';
+import { Remarkable } from 'remarkable';
+
+import { memoizeOne } from 'core/memoizer';
+
+interface IProps {
+    active: boolean;
+    applyFocus: boolean;
+    className: string;
+    value: any;
+}
+
+export default class CellMarkdown extends PureComponent<IProps> {
+    private static md: Remarkable = new Remarkable();
```

Comment:
I've been thinking that maybe we can include the configuration for highlight.js usage, externalize the resource and expect special code block users to include the highlight.js they want as a resource from https://highlightjs.org/download/ ?! Either that or (1) not support it at all or (2) asyncing the resource -- in which case `React.Suspense` won't do and we'll have to handle it through state on `CellMarkdown`. This is mostly about completeness.. we should not load JS by default as it will involve a very special use case.

```
    private static md: Remarkable = new Remarkable({
        highlight: (value: string, language: string) => {
            const hljs: any = (window as any).hljs;

            if (!hljs) {
                return '';
            }

            if (language && hljs.getLanguage(language)) {
                try {
                    return hljs.highlight(language, value).value;
                } catch (_) { }
            }

            try {
                return hljs.highlightAuto(value).value;
            } catch (_) { }

            return '';
        }
    });
```